tests/data/headers/table2_mixed_headers.jsonl

```diff
@@ -0,0 +1,10 @@
+{"EMPLOYEE_ID": 660148, "employee_name": "Michael Roth", "EMPLOYEE_DOB": "2018-06-06", "employee_email": "hsmith@thornton.biz", "fte": 0.610669612273347}
+{"employee_id": 157311, "employee_name": "Angela Moore", "employee_dob": "2008-11-08", "employee_email": "patriciahicks@calderon.org", "fte": 0.40658192760740663}
```

Comment:
I think this should fail?

In terms of a json file I would expect "EMPLOYEE_ID" and "employee_id" to be separate keys and for a table different columns.

What happens if the keys are mixed case but consistent for each json (ie. `"EmpLoYeE_iD":` for every json object in the file)

Reply:
In the first case, you're correct that JSON keys are case sensitive and therefore the test should fail.

In the second case, I think the test _should_ pass, but currently doesn't. I've amended the tests to treat that as correct behaviour, though I think this should be fixed in a future version.